.github/workflows/release.yml

```diff
@@ -29,7 +24,7 @@ jobs:
           python-version: "3.8"
           channel-priority: strict
           activate-environment: dask-sql
-          environment-file: continuous_integration/environment-3.8-jdk11-dev.yaml
+          environment-file: continuous_integration/environment-3.8-dev.yaml
```

Comment:
We won't need to worry about this until it's time to merge into `main`, but probably important to check that we can pip install the outputs of

```
python setup.py sdist bdist_wheel
```

After running through the env setup of this script